projects/js-packages/idc/components/card-fresh/index.jsx

```diff
@@ -82,10 +83,11 @@ const CardFresh = props => {
 								currentHostName,
 								wpcomHostName
 							),
-							{
-								hostname: <strong />,
-							}
-						) }
+						{
+							hostname: <strong />,
```

Comment:
I believe the `hostname` tag provides more context for the translators.
Other consumers can still rely on the `em` tag, and make it bold if needed.

Reply:
> I believe the hostname tag provides more context for the translators.

My case is that consumers may want to make text bold for other texts rather than only site hostnames. For example, the following text makes more sense than use `hostname` tags: 

```
<strong>%1$s</strong> settings, stats, and subscribers will <strong>start fresh</strong>. <strong>%2$s</strong> will keep its data <strong>as is</strong>.
```

Reply:
To do that, you could use the `em` tags allowed in this PR, and style them to make the text bold.
However, if you need to use both `em` and `strong` in the same text, we could add `<strong />` alongside the `<em />` in this PR. Should I do that?

Reply:
> However, if you need to use both em and strong in the same text, we could add <strong /> alongside the <em /> in this PR. Should I do that?

Yes, that would be awesome.

Reply:
Completed